src/dali_backend.cc

```diff
@@ -26,9 +26,43 @@
 
 #include "triton/backend/backend_model.h"
 #include "triton/backend/backend_model_instance.h"
+#include "src/dali_executor/utils/utils.h"
 
 namespace triton { namespace backend { namespace dali {
 
+struct ModelParameters {
+  explicit ModelParameters(common::TritonJson::Value &model_config)
+  {
+    common::TritonJson::Value params;
+    model_config.MemberAsObject("parameters", &params);
+    GetMember(params, num_threads_key, &num_threads);
+  }
+
+ private:
+  template <typename T>
+  void GetMember(
+      common::TritonJson::Value &params,
+      const char* key,
```

Comment:
```suggestion
      const char &key,
```

Reply:
it's a c style string, so it has to be pointer, right?

Reply:
The question is if it has to be a c style string? Just asking.

Reply:
A static const std::string has to be initialized outside of the class, and I thought it's cleaner when all the (future) key strings are listed inside this class.

Reply:
OK